LICENSE

```diff
@@ -178,15 +179,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 HM Revenue & Customs
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Is this supposed to have happened?

Reply:
Ah, this is the appendix. Nevermind! :)

Reply:
Yes, this is the unadulterated Licence, which documents how to apply it to your work.